### PR TITLE
general: fix Flask-Admin, Workflow requirements

### PR DESCRIPTION
**Pull request by hachreak**
* Sets a stable version for Flask-Admin and Workflow
  in setup.py.

Signed-off-by: Leonardo Rossi <leonardo.r@cern.ch>

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 # from invenio/requirements.txt
 #
--e git+git://github.com/mrjoes/flask-admin.git#egg=Flask-Admin-1.0.9.dev0
--e git+git://github.com/inveniosoftware/workflow.git@e41299579501704b1486c72cc2509a9f82e63ea6#egg=workflow
+# -e git+git://github.com/mrjoes/flask-admin.git#egg=Flask-Admin-1.0.9.dev0
```
Comment (jirikuncar) [inline requirements.txt]: you can remove them completely

File: setup.py [hunk 2]
```diff
@@ -43,6 +43,8 @@
     zip_safe=False,
     platforms='any',
     install_requires=[
+        "Flask-Admin>=1.0.9",
```
Comment (jirikuncar) [inline setup.py]: you can remove it too

Comment (hachreak) [inline setup.py]: so, I have to remove flask-admin and workflow from requirements.txt and also from setup.py?

Comment (jirikuncar) [inline setup.py]: yes, and also `workflows` as both packages are installed by invenio package

**Pull request merged**